lib/common/utils.ts

```diff
@@ -11,50 +11,72 @@
  * @suppress {undefinedVars,globalThis,missingRequire}
  */
 
+// issue #989, to reduce bundle size, use short name
```

Comment:
You shouldn't do the job of a compressor.
In fact, you are probably adding entropy.

P.S.: all of these extra imports/exports add up to browser parsing/interpreting time as well

Reply:
Yes, you are right, I have misunderstanding about uglifyjs, I have modified the PR.